src/exceptions/PS-or-PDF-font-exception-20170817.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <exception licenseId="PS-or-PDF-font-exception-20170817" name="PS/PDF font exception (2017-08-17)" listVersionAdded="3.2">
+      <crossRefs>
+        <crossRef>https://github.com/ArtifexSoftware/urw-base35-fonts/blob/master/LICENSE</crossRef>
+        <crossRef>https://github.com/ArtifexSoftware/urw-base35-fonts/blob/65962e27febc3883a17e651cdb23e783668c996f/LICENSE</crossRef>
+      </crossRefs>
+      <notes>Specified to be used with APGL-3.0</notes>
```

Comment:
Now that we've deprecated `AGPL-3.0` do we want to spell out `AGPL-3.0-or-later or AGPL-2.0-only` here?  And do we want to soften this, based on [your][1]:

> I also agree with your markup approach, but I think we can take it a step further and use <alt> for the license name (rather than leaving the happenstance of AGPL-3.0 per this request), and knock AGPL-3.0 out of the exception name and identifier. This may not be technically necessary now, but seems to be the best way to plan for future use in conjunction with other licenses, which it's my understanding the exceptions are intended to support.

Is there anything AGPL-3.0-specific about the wording of the second paragraph?

[1]: https://github.com/spdx/license-list-XML/issues/616#issuecomment-382951343